tools/pythonpkg/src/include/duckdb_python/import_cache/modules/pyarrow_module.hpp

```diff
@@ -37,14 +37,15 @@ struct PyarrowCacheItem : public PythonImportCacheItem {
 public:
 	PyarrowCacheItem()
 	    : PythonImportCacheItem("pyarrow"), dataset(), Table("Table", this),
-	      RecordBatchReader("RecordBatchReader", this) {
+	      RecordBatchReader("RecordBatchReader", this), MessageReader("MessageReader", this) {
```

Comment:
It should say, but it doesn't (my bad), the cache is generated.
The steps are:
1. adjust `tools/pythonpkg/scripts/imports.py`
2. run `python3 tools/pythonpkg/scripts/generate_import_cache_json.py`
3. run `python3 tools/pythonpkg/scripts/generate_import_cache_cpp.py`
4. run `make format-main` (the generator doesn't respect the formatting rules ;))